packages/nux/src/components/inline-tip/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Notice } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export function InlineTip( { isVisible, className, onRemove, children } ) {
```

Comment:
I wonder if it's worth deprecating the DotTip with the introduction of InlineTip. It could always be a separate PR later on.

Also, given how tricky it's been to get the tips experience right, might be worth making these experimental? (that's my solution to everything today 😄 )

Reply:
> Also, given how tricky it's been to get the tips experience right, might be worth making these experimental? (that's my solution to everything today 😄 )

Good idea 😛 e4fcd64718af1729eae49d3701e4fc54270e0923